bigquery_etl/copy_deduplicate.py

```diff
@@ -254,10 +254,28 @@ def _copy_join_parts(client, stable_table, query_jobs):
             job_config = bigquery.CopyJobConfig(
                 write_disposition=bigquery.WriteDisposition.WRITE_TRUNCATE
             )
-            copy_job = client.copy_table(sources, stable_table, job_config=job_config)
+
+            # workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1722546
+            # split copy_table() jobs
+            tmp_table_1 = temp_dataset.temp_table()
+            copy_job_1 = client.copy_table(
+                sources[: len(sources) // 2], tmp_table_1, job_config=job_config
+            )
+            copy_job_1.result()
+
+            tmp_table_2 = temp_dataset.temp_table()
+            copy_job_2 = client.copy_table(
+                sources[len(sources) // 2 :], tmp_table_2, job_config=job_config
+            )
+            copy_job_2.result()
+
+            copy_job = client.copy_table(
+                [tmp_table_1, tmp_table_2], stable_table, job_config=job_config
+            )
```

Comment:
for `sources` each table is formatted as `f"{sql_table_id(job.destination)}${partition_id}"`, does that need to be done here for `[tmp_table_1, tmp_table_2]` ?

Reply:
to be clear, if it works and doesn't erase all of main_v4, I don't think it's necessary. Just making sure this has been checked before merging.

Reply:
at least in my sandbox project I needed this, filed #2228 to fix